Source/NomadApp.cpp

```
@@ -0,0 +1,1710 @@
+    
+    return NUIKC::Unknown;
+}
+
+} // namespace
+
+// =============================================================================
+// Helper Component: ScopeIndicatorLabel
+// =============================================================================
+class ScopeIndicatorLabel : public NomadUI::NUILabel {
+public:
+    ScopeIndicatorLabel(const std::string& text) : NUILabel(text) {
```

Clang (14.0.6)

[warning] constructor does not initialize these fields: m_icon

(cppcoreguidelines-pro-type-member-init)